integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb

```diff
@@ -1,10 +1,11 @@
 require 'git_utils'
 require 'r10k_utils'
 require 'master_manipulator'
-test_name 'CODEMGMT-73 - C63184 - Single Environment Purge Unmanaged Modules'
+test_name 'CODEMGMT-78 - Puppetfile Purge --puppetfile & --moduledir flag usage'
 
 #Init
 master_certname = on(master, puppet('config', 'print', 'certname')).stdout.rstrip
+moduledir = on(master, puppet('config', '--environment', 'production', 'print', 'environmentpath')).stdout.strip + '/production/modules'
```

Comment:
iirc, you were just calling `config print moduledir` before, right? Why didn't that work?

I'm surprised you need to include the `production/` dir in the path you're appending - I thought the `environmentpath` would already include the `production/` dir?

Reply:
Good catch.

Before I was calling moduledir, which isn't a Puppet thing. It's an actually an r10k thing and I was confused, and modulepath could return a ':' separated list of directories to search for modules, so I decided against using that.

I then put in `--environment production print environmentpath` thinking I'd get back something that ended in production. However, this is returning something I thought was called an "environmentspath" but it looks like "environmentspath" isn't a thing. So we have to append the environment name to it. In that case I don't believe the `--environment production` is necessary anymore and can be removed. I'll do that.

Working in file-sync has destroyed my recollection of this part of Puppet's settings, and I was being sloppy because this acceptance angered me so much.

Reply:
haha all good! I've never had an understanding of Puppet's settings, so no worries 😅 
thanks for explaining!